app/controllers/hooks_controller.rb

```diff
@@ -35,4 +36,13 @@ def verify_payload
     end
     # rubocop:enable GuardClause
   end
+
+  def update_last_webhook_recieved(payload_body)
+    github_organization_id = payload_body.dig("organization", "id")
+    return false unless github_organization_id
+    OrganizationWebhook
+      .find_by!(github_organization_id: github_organization_id)
```

Comment:
Chose to use the unsafe `find_by!` here because it is _exceptional_ if we receive a webhook for an organization that isn't on record. 👍 👎 ❓

Reply:
Sounds good to me 👍